Formula/pandoc.rb

```diff
@@ -3,6 +3,7 @@ class Pandoc < Formula
   homepage "https://pandoc.org/"
   url "https://hackage.haskell.org/package/pandoc-2.10.1/pandoc-2.10.1.tar.gz"
   sha256 "938a4c9b0a7ed3de886c73af4052913b0ac9e4aa12b435bd2afd09670bd3229a"
+  license "GPL-2.0"
```

Comment:
Should `BSD-3-Clause` be added as well?

The [license file](https://github.com/jgm/pandoc/blob/1e1c5e7f4ca46676fe8bbdd7545369220b71e9a3/COPYRIGHT#L36-L69) states that:

> The modules in the `pandoc-types` repository (Text.Pandoc.Definition,
> Text.Pandoc.Builder, Text.Pandoc.Generics, Text.Pandoc.JSON,
> Text.Pandoc.Walk) are licensed under the BSD 3-clause license:

Not sure what exactly the `pandoc-types` are and if we use it

Reply:
The `dual license` situation is when upstream stated all the work is released under two licenses. 

And re-license kick in when the work consists of two licenses or multiple licenses internally or externally. 

Then GPL-2.0 or GPL-3.0 are strong copyleft licenses, so when there are multiple licenses in the same project, the less permissive one (GPL-x) would generally win.

Upstream repos would generally license the different modules under different licenses (GPL license or LGPL license for the source code or executable, BSD, LGPL, or even CC licenses for documentation or artwork), on in this case, I believe those types or templates modules are documentation or artwork.

So when combine the two licenses, it would become GPL-x license.

Reply:
Makes sense. Thanks for the explanation